src/xcode/ENA/ENA/Source/Scenes/Vaccination/__tests__/VaccinationQRCodeScannerViewModelTests.swift

```diff
@@ -90,7 +91,7 @@ class VaccinationQRCodeScannerViewModelTests: XCTestCase {
 		)
 
 		viewModel.activateScanning()
-		let metaDataObject = FakeMetadataMachineReadableCodeObject(stringValue: emptyGuid)
+		let metaDataObject = FakeMetadataMachineReadableCodeObject(stringValue: "HC1:\(emptyGuid)")
```

Comment:
`emptyGuid` is a constant. Why not just adding the plain, empty string?

```suggestion
		let metaDataObject = FakeMetadataMachineReadableCodeObject(stringValue: "HC1:")
```

Reply:
Purely just for readability, as we normally has the "HC1:" prefix for verification, in a later test we change this prefix and add a valid id so the test still fails